ccmlib/scylla_repository.py

```diff
@@ -122,3 +126,10 @@ def __get_dir():
     if not os.path.exists(repo):
         os.mkdir(repo)
     return repo
+
+
+def run_scylla_install_script(install_dir, target_dir):
+    subprocess.check_call(['bash', '-c', ''' sed 's|"$prefix|"$root/$prefix|' -i install.sh '''], cwd=install_dir)
+    subprocess.check_call(['bash', '-c', ''' {0}/install.sh --root {1}  --target  centos --disttype redhat --pkg server '''.format(install_dir, target_dir)], cwd=install_dir)
+    subprocess.check_call(['bash', '-c', 'mkdir -p {0}/conf; cp ./conf/scylla.yaml {0}/conf'.format(target_dir)], cwd=install_dir)
```

Comment:
Nit: to be consistent with above command this might better:
```
subprocess.check_call(['bash', '-c', 'mkdir -p {1}/conf && cp {0}/conf/scylla.yaml {1}/conf/'.format(install_dir, target_dir)], cwd=install_dir)
```